content/guides/tanzu-labs-engineer-faq/index.md

```diff
@@ -0,0 +1,364 @@
+
+But how does sorting stories relative to each other help us estimate when something will be done? Because of velocity. Assign points to each story that you've estimated. For example, your team may use a fibonacci pointing sequence: 1, 2, 3, 5, 8. That's five buckets to put stories in. The easiest stories go into the "1" bucket. The hardest stories go into the "8" bucket. Note that teams will also keep in mind estimates from previous iterations. Imagine a really complex story in a previous iteration got an 8, but the hardest story in this iteration isn't as complex. So maybe they put this iteration's hardest story in the "5" bucket.
+
+Next, keep track of how many story points the team delivers week over week (or better yet, let Tracker keep track of it for you). Figure out how many points the team can deliver in a given week by maintaining a rolling average of the last three week's of points delivered each week. That rolling average—the number of points a team can deliver in a week—is velocity.
+
+[Velocity](https://martinfowler.com/bliki/XpVelocity.html) is the key to planning on XP teams. Instead of asking a team to estimate when something will be done based on their gut, we predict when something will be done based on data. We know how many points the team can deliver week over week. And we know where the story is in the prioritized list of stories (i.e., the backlog).
+
+However, it's also important to know that predictions made with velocity are still just that: predictions. They're not foolproof. That's because a team probably does not deliver the exact same amount of points week over week. There's variation each week in how many points they deliver. That variation we call "volatility." The more volatility, the higher the margin of error in velocity-based predictions.
+
+That's why engineers have to keep a very close watch on volatility on their team; when they're experiencing high degrees of volatility, they have to figure out the cause of the volatility and eliminate it. Otherwise, they won't be able to predict when things will be done with confidence—and the PMs won't be able to plan.
+
+## Q: Why Don't We Estimate Bugs and Chores?
+
+Velocity predicts when stories will be done. So why don't we estimate bugs and chores? Because if you do, you won't be able to rely on your velocity to predict when stories will be done anymore.
+
+Here's why: stories are planned, but bugs are an **unintended consequence of feature development** (and one that, as much as possible, we try to eliminate). A bug represents something that used to work, but now doesn't. You already got points for the work when it was a feature. You shouldn't get more points for breaking it and then fixing it again.
```

Comment:
I might avoid framing the last two sentences in terms of getting or not getting points. In my experience, some people confuse 'not getting points' for 'not getting recognized for my hard work'. Maybe instead, this could be "If breaking a feature and then fixing it increased the team's velocity, that would make it harder to use velocity to predict future story delivery."

Reply:
Good point -- will review and rewrite.

Reply:
Updated.